.travis.yml

```diff
@@ -76,6 +76,11 @@ install:
 
   - cd ${TRAVIS_BUILD_DIR}
 
+before_script:
```

Comment:
What do these do?

Reply:
oh i need to get rid of that. old legacy travis changes to try to get headless rendering working.